.gitlab-ci.yml

```diff
@@ -747,23 +759,42 @@ publish-rustdoc:
     - rm -rf .git/config
     - git config user.email "devops-team@parity.io"
     - git config user.name "${GITHUB_USER}"
-    - git config remote.origin.url "https://${GITHUB_TOKEN}@github.com/paritytech/substrate.git"
+    - git config remote.origin.url "https://${GITHUB_TOKEN}@github.com/paritytech/${CI_PROJECT_NAME}.git"
     - git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
     - git fetch origin gh-pages
+    # Install `ejs` and generate index.html based on RUSTDOCS_DEPLOY_REFS
+    - yarn global add ejs
+    - |
```

Comment:
If we combine 767 - 768, yaml doesn't parse the ':' and '"' correctly, or we need some tricky character escaping. So just making this multiline, and parse everything as string.

Reply:
for this you could put the command into single quotes: 
`- 'command'`